pkgs/development/python-modules/tpm2-pytss/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, buildPythonPackage, fetchPypi, pythonOlder
+{ lib, buildPythonPackage, fetchPypi, pythonOlder, python3, python3Packages
```

Comment:
Using python3 or python3Packages is not allowed in pythonPackages because it is mixing versions. Please use the packages directly or python.

Reply:
Thank you for this review! Sorry I am not familiar with buildPythonPackage and don't really understand this suggestion here. Maybe the fixup commit is correct